pytest_order/sorter.py

```diff
@@ -170,8 +178,7 @@ def item_from_label(
                         continue
                     if item_id.startswith(id_start):
                         return self.node_ids[node_id]
-        except KeyError:
-            return
+        return None
```

Comment:
I never use this, as it is unneeded - is there some style guide that demands that?

Reply:
Well, not really. `contextlib.suppress` signifies the exception is specifically being ignored though, instead of it being handled in any special way (like you'd do with a try-except). I can revert it to `try:` and `except KeyError: pass`, if that would be preferable by you instead.

Reply:
I just tried the tests without it entirely, and they still pass. Is there any specific reason this exists there even?

Reply:
What I meant was `return None`, not `contextlib.suppress` - I like that one. As for why it is needed - maybe I'm just missing a test where `last_comp` is not in `node_id_last`.

Reply:
Ah, I see. The reason why I added it there, is because MyPy complained that the function can "just end", without hitting a return statement, which was true, at least in theory (at runtime, it may differ). Also, PEP8 itself recommends adding an explicit `return None` too: https://i.imgur.com/S2XopLp.png

If coverage is the problem there, there should be a way to exclude that particular line from being covered. I'll try adding a `# pragma: no cover` to it, and see what happens.

Reply:
No, that's fine as is - I just didn't remember that from PEP8, obviously missed it. Coverage actually shall count this as a branch, wheather explicitely stated or not - if not, it's a bug in coverage.